main.rb

```diff
@@ -73,6 +79,25 @@ class Saying
   end
 end
 
+put '/send-to-slack' do
+  saying = Saying.find(params[:id])
+  if saying.inc(:crap_count, 1)
+    @attachments = [
+      {
+        author_name: saying.who,
+        color: 'good',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.